src/_pytest/pathlib.py

```diff
@@ -37,10 +37,15 @@ def ensure_reset_dir(path):
 
 
 def rmtree(path, force=False):
```

Comment:
@RonnyPfannschmidt I see that nobody passes `force=False` on the codebase, how about removing this parameter and always assume `force=True`?

Reply:
Go for it

Reply:
Done